File: 백준/Bronze/1546. 평균/평균.java
import java.util.*;

public class Main {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		
		int N = s.nextInt();
		s.nextLine();
		StringTokenizer st = new StringTokenizer(s.nextLine());
		
		double[] arr = new double[N];
		double max = 0;
		for(int i = 0; i < N; i++) {
			arr[i] = Integer.parseInt(st.nextToken());
			max = Math.max(max, arr[i]);
		}
		
		double sum = 0;
		for(int i = 0; i < N; i++) {
			sum += (arr[i] / max * 100);
		}
		
		System.out.println(sum / N);
	}
}

File: 백준/Bronze/1546. 평균/평균.kt
import java.io.*;
fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val N = br.readLine().toInt()

    val list = br.readLine().split(" ")

    var sum = 0.0
    var max = 0.0
    for(i in list) {
        sum += i.toDouble();
        max = maxOf(max, i.toDouble())
    }

    println((sum / max * 100.0)/N);
}